### DISCUSSION TITLE
Screen sometimes goes black on first adjustment

### DISCUSSION THREAD

**Discussion by pocketpixels** [MonitorControl/MonitorControl#1371] [Q&A]
Generally MonitorControl works great for me. I am using the keyboard brightness keys to adjust the brightness of my Dell U3223QE monitor connected via USB-C to my M2 Macbook Air running in clamshell mode. 
However sometimes when I first change the brightness (and I think it has always been me decreasing the brightness), the screen goes fully black. And the only way to recover that I have found so far is to open my laptop and then close it again, returning to clamshell mode. 
Any idea what might be causing this, and is there a setting that might avoid this issue? 
I am currently using hardware DDC control.

**Comment by beefstew** [comment]
I'm seeing maybe a similar behavior, all of a sudden. Until now, the software has worked fantastically!

Any time I control the brightness via Monitor Control, the screen immediate goes black. To recover, I have to power-cycle the monitor -- OSD controls don't even to work in this black mode.

monitor Lenovo p40w-20
laptop MacBook Pro 14-inch, 2021 Apple M1 Pro
macOS Version 13.3.1 (22E261)

I'm using the MacBook in clamshell mode. The monitor is connected directly via USB-C/Thunderbolt cable. 

Sunday, this was working. Today, it's not. I did not change any software (still have a Ventura 13.4 update pending).

![Screenshot 2023-05-24 at 5 52 46 PM](https://github.com/MonitorControl/MonitorControl/assets/2262454/b78e7ed0-5a7e-4cc6-8708-51240919437a)

**Comment by nhchan** [comment]
I have the same issue — black/blank screen on first adjustment. It does not always happen, but when it does, it keeps happening until I give up. This tends to happen in the evening, although I have not (yet) created a record to be scientifically certain of this correlation. 

To recover, I either turn the monitor off and on again, or unplug and re-plug the cable.

**Monitor:** Philips, PHL 346P1C
**Computer:** MacBook Pro 16-inch M1 Max (this also occurs with a 2019 15-inch MacBook Pro with Intel chip; same monitor)
**Connection used:** USB-C
**OS:** macOS 13.5.1 (same issues before also with 12.x.x; no tests with even earlier versions)

I kept hoping that version updates would miraculously (and perhaps accidentally) make this issue go away, but it has been going on for almost two years (since the beginning of my using Monitor Control), so I thought I should be more serious about raising the issue. 

A second monitor connected to the same machine displays none of these symptoms, whether over HDMI or USB-C.

I do **not** use my laptops in clamshell mode, so that may not be the cause here.

**Comment by luistrong** [comment]
Same issue black screen on first adjustment happens on two monitors using MonitorControl v4.1.0 and 4.2.0.
MBA M2, macos12.6.2, 4k60hz monitor, DP-C.
 (mac is not in clamshell mode)

**Comment by antoniomcr96** [comment]
Same issue. Screen goes black on brightness or audio adjustments and then it recovers after a few seconds. MBA M2, HP 24ea monitor connected with a usbc -> hdmi adapter
